assets/messages.yml

```diff
@@ -902,13 +1102,14 @@ messages:
   wai:
     - project: mc
       name: Works As Intended
+      shortcut: wai
       message: |-
         *Thank you for your report!*
         However, this issue is {color:#FF5722}*Working as Intended*{color}.
 
         The report you have submitted is working as intended: %s%.
 
-        Please note, that mechanics of the game may change between updates.
+        Please note that mechanics of the game may change between updates.
         Things such as graphics, sounds, world creation, biomes, redstone, villagers, and animals may not work the same in current versions.
 
         [Full Version History|https://minecraft.gamepedia.com/Java_Edition_version_history] -- [Snapshot Version History|https://minecraft.gamepedia.com/Java_Edition_version_history/Development_versions] -- [Feature Requests and Suggestions|https://www.reddit.com/r/minecraftsuggestions]
```

Comment:
The "Feature Requests and Suggestions" link here (same for mcpe and mcl) is redundant because it is in the quicklinks as well (though there it points to the feedback site).
Should it be removed?

Reply:
I'd say that probably all of these links should be removed, they all don't really seem too relevant to me. But personally I don't use this message anyway.

The link to minecraftsuggestions is there because the link to the feedback site was only added a lot later.

Reply:
I think it would be better to change / discuss this in a seperate issue / pr.

Reply:
Created #87 for this